test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -5728,7 +5728,7 @@ TEST(mongoUpdateContextRequest, updateAttrWithId)
 *
 * updateAttrWithAndWithoutId -
 */
-TEST(mongoUpdateContextRequest, updateAttrWithAndWithoutId)
```

Comment:
Added a fixme to the exact line that causes the crash and left the functest in DISABLED mode, in 9b0269c